### ISSUE TITLE
Crash after calling animateViews in tableView

### ISSUE THREAD

**Issue by jpv123** [marcosgriselli/ViewAnimator#26]
## Expected Behavior
Aninate and not crash. This was working a month ago. I just reopened my project and now I can't get it working

## Actual Behavior
Crash in line 49 of ViewAnimator.swift

## Steps to Reproduce the Problem

  1. Setup a simple tableView
  2. Reload data
  3. Just after reloading data call animate views like this
`tableView.animateViews(animations: [AnimationType.from(direction: .bottom, offset: 30.0)])`

## Specifications

  - Version: 1.0.4 of ViewAnimator
  - Swift 4.0
  - Deployment target is 11.2.

**Comment by jpv123** [comment]
It looks like it's a problem with the completion which is set to

`dispatchGroup.leave()` on Animatable.swift line 74.

**Comment by jpv123** [comment]
@marcosgriselli Any ideas?

**Comment by jpv123** [comment]
This is solved by PR https://github.com/marcosgriselli/ViewAnimator/pull/25. However completion of animateViews becomes automatically executed with this code. Solution (Workaround while it's fixed and there's a new release 1.0.5) was to apply a Dispatch after inside completion

```
DispatchQueue.main.asyncAfter(deadline: .now() + AnimationDuration, execute: {
    //Your completion code            
})
```

Hope this can help those who have the same problem in the future
